python/dune/mmesh/test/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ add_subdirectory(grids)
 
 add_python_targets(test __main__)
 
+if(HAVE_DUNE_FEM)
```

Comment:
I think the CMake-CMake variable is `dune-fem_FOUND`. Not sure if `HAVE_DUNE_FEM` is meant to be available in CMake.

Reply:
Oh yeah, thanks! I will change it.

Interestingly, it actually seems to work with this...